### PR TITLE
OSDOCS-12375: Replace directory structure

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
TBD

Issue:
[OSDOCS-12375](https://issues.redhat.com//browse/OSDOCS-12375)

Link to docs preview:
TOC, many topics in the [Machine management](https://84127--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/) book, and locations linked to from that book

QE review:
No changes to existing content at this stage

Additional information:
**Reviewers**: please reach out directly with questions, I am happy to chat about this.

### CODE REVIEW

File: machine_management/compute_machine_management/mapi-compute-about.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi-compute-about.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi-compute-configuration.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi-compute-configuration.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi-compute-disabling.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi-compute-disabling.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi-compute-getting-started.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi-compute-getting-started.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi-compute-managing-machines.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi-compute-managing-machines.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi-compute-resiliency.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi-compute-resiliency.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi-compute-troubleshooting.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi-compute-troubleshooting.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-aws.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-aws.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-azure.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-azure.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-gcp.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-gcp.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-nutanix.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-nutanix.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-openstack.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-openstack.adoc:1]: placeholder

File: machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-vsphere.adoc
Comment (jeana-redhat) [inline machine_management/compute_machine_management/mapi_compute_provider_configurations/mapi-compute-config-options-vsphere.adoc:1]: placeholder

**Review by jeana-redhat — COMMENTED** [review]
Noting placeholder topics. These are duplicates of similar content to stand in for later work. They are commented out in the TOC and will not appear anywhere.

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -2310,10 +2310,51 @@ Distros: openshift-origin,openshift-enterprise
...
+  #   - Name: Compute configuration options for VMware vSphere
+  #     File: mapi-compute-config-options-vsphere
+  - Name: Applying autoscaling to a cluster
+    File: applying-autoscaling
+  # - Name: Compute resiliency and recovery
+  #   File: mapi-compute-resiliency
+  # - Name: Troubleshooting the compute machine set
+  #   File: mapi-compute-troubleshooting
+  # - Name: Disabling compute machine sets
+  #   File: mapi-compute-disabling
```
Comment (jeana-redhat) [inline _topic_maps/_topic_map.yml:2354]: Commented out assemblies and directories will be uncommented and added in a later stage

File: machine_management/manual_machine_management/user_infra/adding-compute-user-infra-general.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="adding-compute-user-infra-general"]
+= Adding compute machines to clusters with user-provisioned infrastructure manually
+include::_attributes/common-attributes.adoc[]
+:context: adding-compute-user-infra-general
+
```
Comment (mburke5678) [inline machine_management/manual_machine_management/user_infra/adding-compute-user-infra-general.adoc:6]: This assembly is subsequently overwhelming and underwhelming! 
The heading after heading after heading seems like a bit much visually, to me. And then, each module is only a link is a bit disappointing. 
I wonder if there is another way to present this.  Or not. Up to you. Matter of opinion.

Comment (jeana-redhat) [reply machine_management/manual_machine_management/user_infra/adding-compute-user-infra-general.adoc:6]: Yeah, sprucing this assembly up is part of the plan. For now it has moved as-is, but your feedback is right on the money in terms of what I think it needs :clinking_glasses:

**Pull request merged**